js/app.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+
+   $('#submit-btn').click(function(event){ 
+      event.preventDefault();
+      var city = $('#city-type').val();
+   
+    
+	 if (city === 'New York' || city === 'New York City' || city === 'NYC') {
+    // make the page background image of New York City if one of these 3 terms are entered
+      $('body').removeClass(); $('body').addClass('nyc');
```

Comment:
Two things:

1. You are using `.removeClass();` but you aren't actually removing anything. It should be `.removeClass('className');`

2. I would use chaining here, like: `$(body).removeClass('className').addClass('className');`